mina-daemon/templates/deployment.yaml

```diff
@@ -408,10 +424,18 @@ spec:
 {{- include "healthcheck.node.allChecks" $data | indent 10 }}
           imagePullPolicy: Always
           volumeMounts:
-            {{- if .Values.node.libp2pKeys.enabled }}
+            - name: {{ .Release.Name }}-scripts
+              mountPath: /scripts/entrypoint.sh
+              subPath: entrypoint.sh
+            {{ if .Values.node.libp2pKeys.enabled -}}
+            {{ if .Values.node.libp2pKeys.create -}}
```

Comment:
if you're combining enable and create why not have it under one if?
`{{ if and .Values.node.libp2pKeys.enabled .Values.node.libp2pKeys.create -}}`

Reply:
This is because the logic is this one:

if libp2pKeys.enabled
   if libp2pKeys.create
       do xxxxxx # Case where it is autogenerated
   else
       do xxxxxx # Case where we import libp2pKeys from secret
   fi
fi

Reply:
exactly my point, you only get to do the logic if both are true. as a proof you can see at the bottom of your example there is no code/logic between two `fi`s. They both close one after another without providing extra code or `else` for the outer if.

Reply:
Oh I see,
do you suggest
```
if libp2pKeys.enabled and libp2pKeys.create
    do xxxxxx # Case where it is autogenerated
else
    do xxxxxx # Case where we import libp2pKeys from secret
fi

```

Reply:
I am sorry, you are right, I misread the code. I see indeed that if we combine both conditions with `and` and one of them is false, then templating will produce the `else` code even it `enabled` will be disabled and we would not want any code produced. Please disregard my comments and merge at will